src/VirtualClient/VirtualClient.Actions.UnitTests/Examples/Memtier/RedisResults_3.txt

```diff
@@ -0,0 +1,392 @@
+6379
+4         Threads
+1         Connections per thread
+30        Seconds
+
+
+ALL STATS
+============================================================================================================================================================
+Type         Ops/sec     Hits/sec   Misses/sec    Avg. Latency     p50 Latency     p90 Latency     p95 Latency     p99 Latency   p99.9 Latency       KB/sec
+------------------------------------------------------------------------------------------------------------------------------------------------------------
+Sets        35598.74          ---          ---         0.34304         0.33500         0.47900         0.55900         0.83900         1.54300      2742.53
```

Comment:
I don't see an example where you need aggregation, could you add that?

Reply:
Sure. Aggregation is across different output files generated from each client communicating to different Redis server processes running on different ports. Total number of output files generated will be equal to VCPUs*clientInstanceCount. We are aggregating here to know how all the Redis processes are running on a system as a whole. Metrics from individual Redis processes have become a little less important than the overall aggregate metrics. Anyway, I will change unit tests to accept multiple set of results for aggregation. Thanks Yang.

Reply:
Thanks I just want to see a set where it is different